contracts/staking.sol

```diff
@@ -56,20 +68,25 @@ contract Staking {
         emit Staked(msg.sender, msg.value);
     }
 
-    function unstake() public onlyStaker {
+    function _unstake() private {
+        require(
+            _validators.length > MinimumRequiredNumValidators,
```

Comment:
Maybe >=?
```suggestion
            _validators.length >= MinimumRequiredNumValidators,
```

Reply:
@Vuksan 

This condition is making sure that the number of validators doesn't go below `MinimumRequiredNumValidators`. If the number of current validators is `MinimumRequiredNumValidators`, any validator can't unstake.

Reply:
You're right 🙂